resources/views/layouts/default.blade.php

```diff
@@ -426,7 +426,7 @@
                 <!-- sidebar: style can be found in sidebar.less -->
                 <section class="sidebar">
                     <!-- sidebar menu: : style can be found in sidebar.less -->
-                    <ul class="sidebar-menu" data-widget="tree">
+                    <ul class="sidebar-menu" data-widget="tree" style="margin-right: {{ \App\Helpers\Helper::determineLanguageDirection() === 'rtl' ? '12px' : '0px' }};">
```

Comment:
I'd probably do something like this:

` {{ \App\Helpers\Helper::determineLanguageDirection() == 'rtl' ? 'style="margin-right:12px' : '' }};"`

No need to specify a margin at all if rtl isn't true, and this could have other layout effects if we specify it.